lib/reverse_words.rb

```diff
@@ -2,5 +2,32 @@
 # Time complexity: ?
 # Space complexity: ?
 def reverse_words(my_words)
-  raise NotImplementedError
+  return nil if my_words == nil
+  i = 0
+  j = 0
+  while i < my_words.length
+    if my_words[i] != " " && i != nil
```

Comment:
How can i ever be nil? Did you mean my_words[i] != nil? In any case, this looks unnecessary.